### ISSUE TITLE
Not working for other videos

### ISSUE THREAD

**Issue by suhas-goutham** [Deevoluation/ALPR#14]
Hi,
I have used another video to see if it is working.Unfortunately it comes as "no license plates were detected".
In the video, we can clearly see the number plate.
The video I have used is for 4 min.
Should i make any changes in the code

Is there any solution for this?

**Comment by Saqhas** [comment]
No, you don't need to make any changes to the code. The video might be a bit difficult for the detector.
Can you please mail the video to me. I will check it.
Mail id: saquibnadeemhashmi@gmail.com

**Comment by suhas-goutham** [comment]
Sure i can mail it. Thanks a lot !

**Comment by Saqhas** [comment]
The error seems to be with opencv. It is generating the frames from the video but they are automatically rotated 90.
We are looking into this problem. Will get back to you soon.

**Comment by suhas-goutham** [comment]
Thank you.
Can you also check why only 10 frames are generated for each video?
I have tried the code with different images and videos I don't get the correct output.

**Comment by suhas-goutham** [comment]
You can comment two lines in videosplit.py
There it rotates the image.
But even after solving this problem.The accuracy is very low.

**Comment by Siddhant96** [comment]
Hey suhas201227, I have uploaded a new file videosplit_sklearn.py . Also, changed the code in Start.py file to use the functionality of this new file rather than videosplit.py file. Now, there will be no problem of rotation.

**Comment by Siddhant96** [comment]
@suhas201227 , You can change the numerical value of variable "framesWeNeed" to get desired number of frames.(It will generate frames near to the value given and not exact.) The code for video splitting is not robust. We designed it for videos of length upto 10 seconds. It is recommended to use short length videos which are in seconds.

**Comment by suhas-goutham** [comment]
Thanks a lot. I will check it out

**Comment by Siddhant96** [comment]
Feel free to contact further. We will try to get back to you as soon as we can.
